modules/loki-rate-limit-errors.adoc

```diff
@@ -0,0 +1,83 @@
+// Module is included in the following assemblies:
+// * logging/cluster-logging-loki.adoc
+// * logging/cluster-logging-external.adoc
+
+:_content-type: PROCEDURE
+[id="loki-rate-limit-errors_{context}"]
+= Troubleshooting Loki rate limit errors
+
+If the Log Forwarder API forwards a large block of messages to Loki that exceeds the rate limit, Loki generates rate limit (`429`) errors.
```

Comment:
The formulation is grammatically unclear: "that exceeds the rate limit" can apply to a "block of messages" or to "Loki". I am not sure I can think of a better way and things seem more clear in context, but this might cause a problem in a translated version.

Reply:
Updated for clarity